journalpump/senders/rsyslog.py

```diff
@@ -53,6 +59,24 @@ def _init_rsyslog_client(self):
             self.rsyslog_client = None
             time.sleep(5.0)
 
+    def send_heartbeat(self):
+        if not self.rsyslog_client:
+            self._init_rsyslog_client()
+
+        try:
+            self.rsyslog_client.log(
+                facility=self.config.get("heartbeat_facility", 5),
+                severity=self.config.get("heartbeat_severity", 7),
+                timestamp=datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ"),
+                hostname=self.config.get("heartbeat_hostname", socket.gethostname()),
+                program=self.config.get("heartbeat_program", "journalpump"),
+                msg=self.config.get("heartbeat_message", "HEARTBEAT"),
+            )
+        except Exception as ex:  # pylint: disable=broad-except
```

Comment:
Should presumably have `except RSYSLOG_CONN_ERRORS as ex:` block here as well to avoid calling `unexpected_exception` for all random network errors, which could get spammy.